portal-api/src/v1/users/repository.js

```diff
@@ -13,15 +12,19 @@ class UserRepository {
     const hashHex = signInTokenHash.toString('hex');
 
     const userCollection = await db.getCollection('users');
-    return userCollection.updateOne({ _id: { $eq: ObjectId(userId) } }, { $set: { signInToken: { hashHex, saltHex, expiry } } });
+
+    return userCollection.updateOne(
+      { _id: { $eq: ObjectId(userId) } },
+      { $push: { signInTokens: { $each: [{ hashHex, saltHex, expiry }], $slice: -SIGN_IN_LINK.MAX_SEND_COUNT } } },
```

Comment:
While not wanting to do this while on a bug fix, we should later be able to remove `signInCount` in favour of this.